.github/workflows/compute.yml

```diff
@@ -46,9 +46,9 @@ jobs:
         run: yarn tsx ./src/main.ts
         id: command-ask
         env:
-            SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
-            SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
-            VOYAGEAI_API_KEY: ${{ secrets.VOYAGEAI_API_KEY }}
-            OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
-            OPENROUTER_API_KEY: ${{ secrets.OPENROUTER_API_KEY }}
-            UBIQUITY_OS_APP_NAME: ${{ secrets.UBIQUITY_OS_APP_NAME }}
\ No newline at end of file
+          SUPABASE_URL: ${{ secrets.SUPABASE_URL }}
+          SUPABASE_KEY: ${{ secrets.SUPABASE_KEY }}
+          VOYAGEAI_API_KEY: ${{ secrets.VOYAGEAI_API_KEY }}
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+          OPENROUTER_API_KEY: ${{ secrets.OPENROUTER_API_KEY }}
+          UBIQUITY_OS_APP_NAME: ${{ secrets.UBIQUITY_OS_APP_NAME }}
```

Comment:
I'm seeing this is used in a lot of plugins so I think it would make sense that it gets passed to the plugin from the kernel

Reply:
Or exported from the SDK maybe if it already exports bot id and node_id I think you said?

Reply:
yeah it exports app id and bot id, but the problem is that it will change in prod/beta or your own kernel

Reply:
valid point that would be more of a nuisance, kernel passing it is far cleaner.